src/components/progress-flow/progress-flow.types.ts

```diff
@@ -0,0 +1,8 @@
+import { ListItem } from '@limetech/lime-elements';
+
+export interface FlowItem extends ListItem {
+    isOffProgress?: boolean;
+    activeColor?: string;
+    passedColor?: string;
+    iconColor?: string;
```

Comment:
These should be better documented since they are part of the public interface :slightly_smiling_face:

Reply:
Sure, let's add some.

Reply:
```suggestion
	/**
     * determines whether a step should be a part of the flow,
     * or should be an independent step visually located on the side. 
     */
    isOffProgress?: boolean;
    
    /**
     * Background color of active step.
     */
    activeColor?: string;
    
    /**
     * Background color of the step, when it is passed or done.
     */
    passedColor?: string;
    
    /**
     * Fill color of the icon on the step, when it is inactive.
     */
    iconColor?: string;
```

Reply:
How about these?